lang/cs/Org.Apache.REEF.Network/Org.Apache.REEF.Network.csproj

```diff
@@ -164,6 +198,7 @@ under the License.
     <Compile Include="Utilities\Utils.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
I believe we have a central one for these. Also, Network is not an app and should not need or be able to use an `app.config`.